crates/rpc/api/src/lib.rs

```diff
@@ -82,6 +82,9 @@ pub trait StrataApi {
     #[method(name = "getCurrentDepositById")]
     async fn get_current_deposit_by_id(&self, deposit_id: u32) -> RpcResult<RpcDepositEntry>;
 
+    #[method(name = "getWithdrawalIntent")]
+    async fn get_current_withdrawal_intent(&self) -> RpcResult<Vec<RpcWithdrawalIntent>>;
```

Comment:
Reflecting the other comment, this should be `getCurWithdrawalAssignments` and `get_cur_withdrawal_assignments`.